exonum/src/helpers/fabric/internal.rs

```diff
@@ -29,9 +29,19 @@ pub enum Feedback {
 
 /// `Command` trait is used to create sub-command for `NodeBuilder`.
 pub trait Command {
+    /// List of command arguments.
     fn args(&self) -> Vec<Argument>;
+    /// Command name.
     fn name(&self) -> CommandName;
+    /// Short information about the command.
     fn about(&self) -> &str;
+    /// Performs command handling code.
+    ///
+    /// # Arguments:
+    ///
+    /// * `commands` - contains list of commands that can be performed on the node.
+    /// * `context` - contains command parameters.
```

Comment:
Is it really `command parameters`? I thought it's more like global execution context.